src/liquid/components/ld-select/ld-select.tsx

```diff
@@ -1304,7 +1502,12 @@ export class LdSelect implements InnerFocusable {
           >
             <div
               ref={(el) => (this.internalOptionsContainerRef = el)}
-              innerHTML={this.internalOptionsHTML}
+              innerHTML={sanitize(this.internalOptionsHTML, {
+                ...(typeof this.sanitizeConfig === 'string'
+                  ? JSON.parse(this.sanitizeConfig)
+                  : this.sanitizeConfig),
+                ADD_ATTR: ['prevent-deselection'],
```

Comment:
What does this line do / why do you add the `prevent-deselection` attribute here?

Reply:
We sanitize the HTML because we don't have control over the options' content. DOMPurify removes attributes it doesn't know, including `prevent-deselection`. So we add the `prevent-deselection` attribute to DOMPurify's default whitelist.